.github/workflows/stale.yml

```
@@ -25,7 +25,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/stale@v8
+    - uses: actions/stale@v9.0.0
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)